gitpractice/javascript/task3.js

```diff
@@ -0,0 +1,19 @@
+/*http://www.codewars.com/kata/basic-mathematical-operations*/
+function basicOp(operation, value1, value2) {
+   var operand = operation;
```

Comment:
You can directly use str, no need to assign it to other variable and use it

Reply:
no need to reassign var operand = operation
you can directly use operation in your code

Reply:
yes i changed it